src/contracts/GPv2Settlement.sol

```diff
@@ -54,6 +54,17 @@ contract GPv2Settlement {
     /// executed.
     mapping(bytes => uint256) public filledAmount;
 
+    /// @dev Event emitted for each executed trade.
+    event Trade(
+        address indexed owner,
+        IERC20 sellToken,
+        IERC20 buyToken,
+        uint256 sellAmount,
+        uint256 buyAmount,
+        uint256 feeAmount,
+        bytes orderUid
```

Comment:
shouldn't this one also be indexed?

Reply:
I left it out pending the discussion on Slack.

Reply:
Created https://github.com/gnosis/gp-v2-contracts/issues/243 so we don't forget to come back to this.